modules/local/eggnog/download.nf

```diff
@@ -26,23 +29,7 @@ process EGGNOG_DOWNLOAD {
         -y \\
         --data_dir ./eggnog/
 
-    cat <<-END_VERSIONS > versions.yml
-    "${task.process}":
-        eggnog: \$( echo \$(emapper.py --version 2>&1)| sed 's/.* emapper-//' )
-    END_VERSIONS
-
-    """
-
-    stub:
```

Comment:
Is the stub not needed anymore? Is it possible to run the download without downloading the data then? E.g. in a test?

Reply:
I get it now. you're right I should leave the stub there. I will put it back :)

Reply:
just restored the stub script